dialect/sql/builder.go

```diff
@@ -2892,10 +2893,11 @@ func (n Queries) Query() (string, []interface{}) {
 type Builder struct {
 	sb        *strings.Builder // underlying builder.
 	dialect   string           // configured dialect.
-	args      []interface{}    // query parameters.
-	total     int              // total number of parameters in query tree.
-	errs      []error          // errors that added during the query construction.
-	qualifier string           // qualifier to prefix identifiers (e.g. table name).
+	tz        *time.Location
```

Comment:
I couldn't find a better place to put this to have it accessible in the predicates, but I think it could be considered similar to `dialact` so it's not a terrible place to put this